cogs/spawning.py

```diff
@@ -228,9 +228,7 @@ async def spawn_pokemon(self, channel, species=None, incense=None, redeem=False)
         else:
             embed.title = "A wild pokémon has appeared!"
 
-        prefix = await self.bot.get_cog("Bot").determine_prefix(channel.guild)
-        prefix = prefix[0]
-        embed.description = f"Guess the pokémon and type `{prefix}catch <pokémon>` to catch it!"
+        embed.description = f"Guess the pokémon and type `{self.bot.user.mention}catch <pokémon>` to catch it!"
```

Comment:
This will show up as `<@id>` since it's in a code block. Can you make this of the form `@{self.bot.user}`?

Reply:
Done 👍